docs/semgrep-supply-chain/ignoring-deps.md

```diff
@@ -15,8 +15,8 @@ You can restrict code files or manifest files or lockfiles from generating Suppl
 | Goal | Method |
 | ---- | ------ |
 | Prevent a code file from generating **any reachable findings**. | Include the code file's path in the repository's `.semgrepignore` file. |
-| Prevent a manifest file or lockfile from generating **any unreachable findings** but still generate reachable findings from a code file. | Include the manifest file's path or the lockfile's path in the repository's `semgrepignore` file. |
-| Prevent a code file from generating either reachable or unreachable findings. | Include the file paths of the manifest file or lockfile and code files in the repository's `.semgrepignore` file. |
+| Prevent any findings from being generated using the dependencies in a manifest file or lockfile | Include the file paths of the manifest file or lockfile in the repository's `.semgrepignore` files |
```

Comment:
```suggestion
| Prevent any findings from being generated using the dependencies in a manifest file or lockfile | Include the file paths of the manifest file or lockfile in the repository's `.semgrepignore` files |
```

I'm unsure if we want to make this plural yet - maybe better for a separate update. But I don't feel strongly about it.

Reply:
oh you're right, that was a type